generators/add-extension/index.js

```diff
@@ -132,7 +132,9 @@ class Generator extends Base {
       if (!this.fs.exists(source)) {
         source = this.templatePath('template.tmpl.py');
       }
-      this.fs.copyTpl(source, target, d);
+      this.fs.copyTpl(source, target, Object.assign({
```

Comment:
please extract the enhanced config argument to an object since it is now defined twice, once for web and once for the server. Next time I will again forget it to change.